src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import f from './Footer.module.css'
+const Footer = () => {
+        return <header className={f.footer}>
+        <div align={'right'}>footer</div>
```

Comment:
Strings might be put in "html" style instead of js expression:
`<div align="right">footer</div>`

Reply:
fix